cli/cmd/get.go

```diff
@@ -41,6 +41,10 @@ import (
 	"github.com/cortexlabs/cortex/pkg/operator/api/userconfig"
 )
 
+const (
+	maxClassesToDisplay = 20
```

Comment:
This could be bigger I think, since it's not a matter of efficiency, correct? Maybe 100?

Reply:
Edit: I don't quite understand how this is enforced, lets discuss